docs/Makefile

```diff
@@ -29,12 +29,11 @@ clean:
 html:
 	@set -e
 	$(SPHINXAPIDOC) -f -F -e -H "OpenDP" -A "The OpenDP Project" -V $(VERSION) -o source/api/python ../python/src/opendp --templatedir source/_templates
-	# TODO: Figure out how to link locally generated Rust docs into sphinx structure.
-	# 	cd ../rust && $(CARGO) doc --no-deps --target-dir ../docs/source/api/rust
 	$(SPHINXBUILD) $(SPHINXOPTS) -D version=$(VERSION) -D 'html_sidebars.**'=sidebar-nav-bs.html source $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+# In contrast to https://docs.rs/opendp, this will document all functions, including private functions.
```

Comment:
docs.rs does document private items because of this:
https://github.com/opendp/opendp/blob/main/rust/Cargo.toml#L81-L89

Reply:
Thanks -- Maybe I was trying to find the corresponding docs for a function that is in `main` but hasn't been released yet? Anyway, less confused now. Will change.